packages/build-tools/src/steps/utils/expoUpdates.ts

```diff
@@ -29,15 +29,23 @@ export async function configureEASUpdateIfInstalledAsync({
   inputs: {
     runtimeVersion?: string;
     channel?: string;
+    throwIfNotConfigured: boolean;
   };
   appConfig: ExpoConfig;
 }): Promise<void> {
   const expoUpdatesPackageVersion =
     await getExpoUpdatesPackageVersionIfInstalledAsync(workingDirectory);
   if (expoUpdatesPackageVersion === null) {
-    throw new Error(
-      `Cannot configure Expo Updates because the expo-updates package is not installed.`
-    );
+    if (inputs.throwIfNotConfigured) {
+      logger.error(
+        'Cannot configure EAS Update because the expo-updates package is not installed.'
+      );
+      throw new Error(
+        'Cannot configure EAS Update because the expo-updates package is not installed.'
+      );
```

Comment:
Why do we need to both log an error and throw an error? Shouldn't throwing be sufficient?